draft-jholland-quic-multicast.md

```diff
@@ -689,7 +715,101 @@ Not permitted:
+
+However, since the client-initiated Stream ID used for the Session ID is a variable-length integer with 4 possible sizes (1, 2, 4, or 8 octets), clients will need the shared data in the stream to be at one of 4 different possible stream offsets in order to process it.
+Hence, for WebTransport as currently specified servers would need to run up to 4 separate channels instead of a single channel in order to send the same WebTransport data to many different clients.
+
+WebTransport Datagrams are delivered over HTTP/3 Datagrams as defined in {{I-D.draft-ietf-masque-h3-datagram}} and have the same characteristic of relying on the client-chosen Quarter Stream ID value.
+
+While using 4 multicast channels instead of 1 still represents a potentially vast scalability improvement over unicast delivery for popular content, it causes other scalability problems, especially in networks that have small limits on the number of multicast channels that are allowed to be provisioned at the same time.
+The total channel limits have a surprisingly low bound in a lot of multicast-capable networking devices designed for TV services that were expected to support only up to a few tens of popular channels at the same time.
+
+It is therefore hoped that an extension or revision to WebTransport and HTTP/3 Datagrams can be adopted in a future version of their specifications that make it possible to use a single channel for all the shared data.
+
+For example, an extension that permits a server-chosen value to be used as a Session ID.
+Such a value could for instance be sent in an HTTP/3 response header, and as long as it is unique within the connection and avoided collision with any client-initiated stream ID values, could still be used to multiplex data associated with different HTTP/3 traffic and different WebTransport sessions carried on the same connection.
+Then by choosing the same server-chosen session ID for all the connections, it would allow the server to use the same channel to carry the identical data, including the Session ID, to be received by multiple receivers.
+Such a change could either replace the current client-chosen definition for Session ID, or could add new frame types that allow a server-chosen Session ID when the client has advertised support for this extended functionality.
+
+As an alternate example of an extension, a mechanism that allows padding at the beginning of the WebTransport stream before the Session ID, for example with 0 bytes, would at least allow the server to align the client-chosen ID value to end at the same QUIC stream offset, which would allow the shared portion of the stream data (all or most of the data following the Session ID) to be transmitted via the same multicast channel, since it would then be the same at the same stream offset.
+
+### Non-web Applications
+
+There are also several non-web application protocols that could profitably use multicast QUIC.  A few examples include:
```

Comment:
I am no expert in language of IETF documents, but "profitably" seems off, maybe better to use something like "efficiently"?